src/Aspire.Dashboard/Components/Pages/Login.razor.js

```diff
@@ -1,6 +1,6 @@
 export async function validateToken(token) {
     try {
-        var url = `/api/validatetoken?token=${encodeURIComponent(token)}`;
+        var url = `api/validatetoken?token=${encodeURIComponent(token)}`;
```

Comment:
We're doing `./xxx` everywhere else. Should this have no slash at the start or `./api/...` to be consistant?

Reply:
If should likely be `./api/...` assuming the JS module loading resolution still works. I can't remember if I tried that. Note the module reference paths from .js files was one of the trickiest things to figure out but I'll try it here.